NCARDDatabase/settings.py

```diff
@@ -128,8 +132,10 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
-STATIC_URL = 'static/'
+STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+STATIC_DIR    = BASE_DIR.joinpath('static')
```

Comment:
What is STATIC_DIR for? It doesn't appear to be a standard Django setting.

If it's only used to fill in STATICFILES_DIR and is not a standard Django setting, give it a lowercase name.

Reply:
I just delete this part in my branch